### PR TITLE
refs #26741 - adds usage / build instructions for foreman container

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Review by Ron-Lavi — COMMENTED** [review]
Thanks @ohadlevy ! works great,
For users who doesn't have docker installed yet,
there needed to add two steps after installing docker:
- need to add the current user to the "docker" group
- than docker-compose pull

**Review by Ron-Lavi — APPROVED** [review]
I saw you just added those points,
thanks for doing this LGTM :+1:

**Comment by ohadlevy** [comment]
> Thanks @ohadlevy ! works great,
> For users who doesn't have docker installed yet,
> there needed to add two steps after installing docker:
> 
> * need to add the current user to the "docker" group
> * than docker-compose pull

@Laviro thanks,I've updated the doc - can you have another look?

**Review by Ron-Lavi — DISMISSED** [review]

File: developer_docs/containers.asciidoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+[building]
+== Building your own Image
+
+[quay]
+=== Using quay.io
+I’ve found its easiest simply to configure https://quay.io to build the images for me, the workflow is simple enough (e.g. on every git push to my repository an image will be built) and it doesn’t slow down my machine while building it (it takes about 7-8 minutes or so).
+
+=== Building locally
+
+Please note that the version of docker that is used in Fedora is too old (doesn't support multi stage builds), one might consider using podman, moby or official docker repository to build locally.
```
Comment (ShimShtein) [inline developer_docs/containers.asciidoc]: Can you please specify the lowest supported version? It should be `Docker 17.05` according to some blog I found.

**Review by ShimShtein — CHANGES_REQUESTED** [review]
Awesome! I like the idea that this doc will live with the code it describes.
Can we also add a reference to here from README for the sake of better visibility?

File: developer_docs/containers.asciidoc (hunk 1 above)
Comment (ohadlevy) [reply developer_docs/containers.asciidoc]: updated.

**Comment by ohadlevy** [comment]
> Awesome! I like the idea that this doc will live with the code it describes.
> Can we also add a reference to here from README for the sake of better visibility?

TBH I'm not sure under which section to put it, I don't want to set the wrong expectation this is the way users should install it, perhaps in the developer handbook?

**Comment by ohadlevy** [comment]
anything left for this PR ? thanks.

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
Let's think about links t this docs later. Merging as is.

**Comment by ShimShtein** [comment]
Merged, thanks, @ohadlevy!

**Pull request merged**